cmake/templates/pkgConfig.cmake.in

```diff
@@ -123,6 +123,25 @@ foreach(library ${libraries})
     list(APPEND @PROJECT_NAME@_LIBRARIES ${library})
   elseif(${library} MATCHES "^-l")
     list(APPEND @PROJECT_NAME@_LIBRARIES ${library})
+  elseif(${library} MATCHES "^-")
+    # This is a linker flag/option (like -pthread)
+    # There's no standard variable for these, so create an interface library to hold it
+    if(NOT @PROJECT_NAME@_NUM_DUMMY_TARGETS)
+        set(@PROJECT_NAME@_NUM_DUMMY_TARGETS 0)
```

Comment:
Nitpick: two space indention.

Same below.

Reply:
Two space indentation in 112e22802c0f0cee150c6484abbaeb55becaa132